srcpkgs/python3-tasklib/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-tasklib'
+pkgname=python3-tasklib
+version=2.4.3
+revision=1
+wrksrc="${pkgname#*-}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-wheel python3-pip task"
+depends="python3 task"
```

Comment:
pytz and python3-tzlocal are missing

Reply:
Is there info on the PyPi site that gives dependencies? All I could gather was that taskwarrior and python are needed ([src](https://pypi.org/project/tasklib/))

Reply:
Usually setup.py, setup.cfg or pytest.ini contains install_requires or an equivalent, also checks will fail if you forget to specify a dependency and don't add pip to checkdepends (which defeats the purpose of having checks).
https://github.com/GothenburgBitFactory/tasklib/blob/f7672ddebe6d4f4ff8fdb743df0b2939ba66ec78/setup.py#L25